core/types/calls.cc

```diff
@@ -1764,7 +1764,8 @@ void applySig(const GlobalState &gs, const DispatchArgs &args, DispatchResult &r
          ++loc) {
         callLocsArgs.emplace_back(*loc);
     }
-    CallLocs callLocs{args.locs.file, args.locs.call, callLocsReceiver, callLocsArgs};
+    // TODO(jez) Double check this
+    CallLocs callLocs{args.locs.file, args.locs.call, callLocsReceiver, args.locs.fun, callLocsArgs};
```

Comment:
Did you get a chance to address this TODO? Not quite sure what it's referring to.

Reply:
yeah, when we rewrite sig to `Sorbet::Private::Static.sig`, we re-use all parts of the `Send` node, and only update the `recv` and `args` field, so the `funLoc` is unchanged.